### ISSUE TITLE
ethereumjs-vm constructor problem with 'null' parameter

### ISSUE THREAD

**Issue by cgewecke** [MetaMask/provider-engine#96]
Hi. Getting the error below on a build that uses testrpc and ran OK a day ago:
```
Error: TypeError: Cannot read property 'state' of null
    at new VM (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/node_modules/ethereumjs-vm/lib/index.js:31:15)
    at VmSubprovider.runVm (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/subproviders/vm.js:84:22)
    at VmSubprovider.handleRequest (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/subproviders/vm.js:47:8)
    at next (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:95:18)
    at /usr/local/lib/node_modules/ethereumjs-testrpc/lib/subproviders/gethdefaults.js:31:7
    at Web3ProviderEngine._inspectResponseForNewBlock (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:231:12)
    at /usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:131:14
    at /usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/node_modules/async/dist/async.js:356:16
    at replenish (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/node_modules/async/dist/async.js:877:25)
    at iterateeCallback (/usr/local/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/node_modules/async/dist/async.js:867:17)
```

I think this is because of changes to [ethereumjs-vm's constructor](https://github.com/ethereumjs/ethereumjs-vm/blob/master/lib/index.js#L29) which now looks like this: 
```javascript
function VM (opts = {}) {
  this.stateManager = new StateManager({
    trie: opts.state,
    blockchain: opts.blockchain
  })
```
causing [this line](https://github.com/MetaMask/provider-engine/blob/master/subproviders/vm.js#L84) in subproviders/vm.js to stop working
```javascript
// create vm with state lookup intercepted
var vm = self.vm = new VM(null, null, {
  enableHomestead: true
})
```

**Comment by flyswatter** [comment]
That constructor [was changed in August](https://github.com/ethereumjs/ethereumjs-vm/commit/7d3f0b8fc7899d94678c330e6587c6ae2dcc9b16), and before it merely didn't have the default empty object.

It looks like the VM subprovider [has done that since April](https://github.com/MetaMask/provider-engine/commit/bcc101b75212fc19175d827112f369d225573230), so I'm surprised that these two nulls ever worked.

That said, I see there was a `web3-provider-engine` version bump just 3 hours ago, while the vm is a week old right now, so since you say it worked yesterday, I'd peg it on a recent change.  I'll point this out to @kumavis.

**Comment by cgewecke** [comment]
Cool thanks!

**Comment by kumavis** [comment]
@cgewecke should be fixed in `8.1.9`. let me know if its not.

**Comment by cgewecke** [comment]
Yes, works. Thanks so much!
